pkg/controller/nodeipam/ipam/multi_cidr_range_allocator.go

```diff
@@ -802,7 +874,7 @@ func (r *multiCIDRRangeAllocator) allocatedClusterCIDR(node *v1.Node) (*cidrset.
 // P2: ClusterCIDR with a PerNodeMaskSize having fewer IPs has higher priority.
 // P3: ClusterCIDR having label with lower alphanumeric value has higher priority.
 // P4: ClusterCIDR with a cidrSet having a smaller IP address value has a higher priority.
-func (r *multiCIDRRangeAllocator) orderedMatchingClusterCIDRs(node *v1.Node) ([]*cidrset.ClusterCIDR, error) {
+func (r *multiCIDRRangeAllocator) orderedMatchingClusterCIDRs(node *v1.Node, occupy bool) ([]*cidrset.ClusterCIDR, error) {
```

Comment:
what does occupy means here?
can you add a comment with the explanation?

Reply:
Added an explanation